apps/integration/integration/automation/runtests.sh

```diff
@@ -33,20 +55,73 @@ checkout_main_if_new() {
 
 build_containers() {
   echo "Yep, definitely building containers. That's what this function does" >&2
+  docker-build-hub apps/crawler/crawler/http/Dockerfile
+  docker-build-hub apps/crawler/crawler/s3/Dockerfile
+  docker-build-hub apps/importer/Dockerfile.buildx
+  docker-build-hub apps/opensearch/Dockerfile
+  docker-build-hub apps/jupyter/Dockerfile.buildx --build-arg=TAG="${TAG}"
+  docker-build-hub apps/demo-ui/Dockerfile.buildx
+  docker-build-hub apps/remote-processor-service/Dockerfile.buildx
 }
 
 handle_outputs() {
   echo "Yep, definitely handling test outputs. That's what this function does" >&2
+  local passed_tests="$1"
+  mv test-output.log "${QUERY_LOGFILE}"
+  [[ ${passed_tests} = 0 ]] && touch "${RUNDIR}/passed"
+  [[ ${passed_tests} != 0 ]] && touch "${RUNDIR}/failed"
+  aws s3 cp -r "${RUNDIR}" "s3://sycamore-ci/${ARCH}"
 }
 
 runtests() {
   docker system prune -f --volumes
```

Comment:
This seems risky since if someone runs it it will prune other stuff they might want; It also means that we get rid of the old volumes; it's also not guaranteed to clean things up if it's still in use.

I'd instead suggest that we generate a unique run id (use the date stamp?). and do the docker compose -p <uid> up. If we tag the images in the same way, then if there's a problem someone could go back and debug against the exact set of images. This would imply we don't want to push to docker hub since it's a lot of stuff.
That guarantees you're getting